pyvisgraph/visible_vertices.py

```diff
@@ -129,6 +129,7 @@ def polygon_crossing(p1, poly_edges):
         co0 = (ccw(p1, edge.p1, p2) == 0) and (edge.p1.x > p1.x)
         co1 = (ccw(p1, edge.p2, p2) == 0) and (edge.p2.x > p1.x)
         co_point = edge.p1 if co0 else edge.p2
+        if co0 and co1: continue

```

Comment:
Lets move this one line up, co_point doesnt need to run when co0 and co1 are True. Save some running time